pkg/core/native/native_neo.go

```diff
@@ -23,6 +23,8 @@ import (
 type NEO struct {
 	nep5TokenNative
 	GAS *GAS
+
+	validators keys.PublicKeys
```

Comment:
I think it should be protected or made atomic-swappable. Contract's methods can be test-invoked from RPC, for example.

Reply:
Yeah, haven't thought about that. Fixed.